conans/test/integration/command/upload/syncronize_test.py

```diff
@@ -15,7 +15,6 @@ class SynchronizeTest(unittest.TestCase):
 
     def test_upload(self):
         client = TestClient(servers={"default": TestServer()}, inputs=["admin", "password"])
-        save(client.cache.default_profile_path, "")
```

Comment:
Why removing this is necessary? It seems only export + upload are used here, which should never need the profile at all?

Reply:
The line 73 is doing `client.run("install --requires=%s --build missing" % str(ref))` so building the package.